tests/0081-admin.c

```diff
@@ -3652,239 +3652,335 @@ static void do_test_ListConsumerGroupOffsets(const char *what,
-                size_t num_results = rd_kafka_AlterUserScramCredentials_result_get_count(result); 
-                size_t i;
-                /* num_results should be 1*/
-                TEST_ASSERT(num_results==1,"There should be only 1 Result for a User !");
-                for (i = 0; i < num_results; i++){
-                        rd_kafka_UserScramCredentialAlterationResultElement_t *element = rd_kafka_AlterUserScramCredentials_result_get_element(result,i);
-                        char *username;
-                        rd_kafka_error_t *error;
-                        username = rd_kafka_UserScramCredentialAlterationResultElement_get_user(element);
-                        error = rd_kafka_UserScramCredentialAlterationResultElement_get_error(element);
-                        rd_kafka_resp_err_t errorcode = rd_kafka_error_code(error);
-                        /* username should be broker and errorcode should be 0*/
-                        TEST_ASSERT(strcmp(users[0],username)==0,"Username does not match !");
-                        TEST_ASSERT(errorcode==0,"Error Code should be 0");
-                }
+        err   = rd_kafka_event_error(event);
+        TEST_ASSERT(err == RD_KAFKA_RESP_ERR_NO_ERROR,
+                    "Expected NO_ERROR, not %s", rd_kafka_err2name(err));
+
+        alter_result = rd_kafka_event_AlterUserScramCredentials_result(event);
+        alter_responses = rd_kafka_AlterUserScramCredentials_result_responses(
+            alter_result, &response_cnt);
+        /* response_cnt should be 1*/
+        TEST_ASSERT(response_cnt == 1,
+                    "There should be exactly 1 response, got %" PRIusz,
+                    response_cnt);
+        for (i = 0; i < response_cnt; i++) {
+                const rd_kafka_AlterUserScramCredentials_result_response_t
+                    *response = alter_responses[i];
+                const char *username;
+                const rd_kafka_error_t *error;
+                username =
+                    rd_kafka_AlterUserScramCredentials_result_response_user(
+                        response);
+                error =
+                    rd_kafka_AlterUserScramCredentials_result_response_error(
+                        response);
+                rd_kafka_resp_err_t err = rd_kafka_error_code(error);
+                /* username should be the same and err should be NO_ERROR*/
+                TEST_ASSERT(strcmp(users[0], username) == 0,
+                            "Username should be %s, got %s", users[0],
+                            username);
+                TEST_ASSERT(err == RD_KAFKA_RESP_ERR_NO_ERROR,
+                            "Error code should be NO_ERROR, got %s",
+                            rd_kafka_err2name(err));
         }
+        rd_kafka_event_destroy(event);
 
-        options = rd_kafka_AdminOptions_new(rk, RD_KAFKA_ADMIN_OP_DESCRIBEUSERSCRAMCREDENTIALS);  
+#if !WITH_SSL
```

Comment:
nit: not needed to wrap this in ifdef, since having the label here isn't a problem for the non-ifdef code also.

Reply:
there is a warning in that case